src/components/common/modal/bulk-edit-modal.styles.tsx

```diff
@@ -0,0 +1,8 @@
+import styled from "styled-components";
+
+export const Wrapper = styled.div`
+    display: flex;
+    flex-direction: column;
+    gap: 16px;
```

Comment:
use `rem` as the gap should scale according to the font size

```suggestion
    gap: 1rem;
```